### PR TITLE
refactor(batch): create Batch samples

### PR DESCRIPTION
**Pull request by ssvir**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by Sita04 — CHANGES_REQUESTED** [review]
PTAL at the comments and the following exception:

`java.lang.NullPointerException: Cannot invoke "java.io.PrintStream.printf(String, Object[])" because "java.lang.System.out" is null`

File: batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  public static void cleanUp() {
+    for (Job job : ACTIVE_JOBS) {
+      try {
+        Util.waitForJobCompletion(job);
+      } catch (IOException | InterruptedException e) {
+        System.err.println(e.getMessage());
+      }
+    }
+    try (DisksClient client = DisksClient.create()) {
+      client.deleteAsync(PROJECT_ID, ZONE, PERSISTENT_DISK_NAME).get(60, TimeUnit.SECONDS);
```
[the remark this answers is not in the document]
Comment (ssvir) [reply batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java:86]: I tried to get  NEW_PERSISTENT_DISK_NAME here, and it throws a not_found exception, maybe the disk deletes after the job finished

**Comment by ssvir** [comment]
> PTAL at the comments and the following exception:
> 
> `java.lang.NullPointerException: Cannot invoke "java.io.PrintStream.printf(String, Object[])" because "java.lang.System.out" is null`

I'm not touching system out. Do you mean that "java.lang.System.out" is null from previous test classes?

**Review by Sita04 — APPROVED** [review]
A couple nits. Kindly fix the lint error before merging.

**Pull request merged**